app/app/src/main/java/de/stustanet/stustapay/ui/NavViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package de.stustanet.stustapay.ui
+
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import de.stustanet.stustapay.data.NfcState
+import kotlinx.coroutines.flow.*
+import javax.inject.Inject
+
+@HiltViewModel
+class NavViewModel @Inject constructor(
```

Comment:
this should not be a navviewmodel, but rather something like `NFCScanState`, which is only used by the nfc scanning view, and not the whole navigation view. this is then used by `ChipStatusView` and triggerd by e.g. a button there?